### PR TITLE
Add preview pane backgrounds for Theater, Playground, and Neighborhood levels

### PR DESCRIPTION
**Pull request by sanchitmalhotra126**
Adds preview backgrounds for Theater, Playground, and Neighborhood. These are longer than they are tall so they'll appear on the sides of the actual content (which is a square) if given the space.

Neighborhood:

Screen Shot 2021-09-27 at 1 44 03 PM

Theater (no content):

Screen Shot 2021-09-27 at 1 44 47 PM

Theater (with content):

Screen Shot 2021-09-27 at 1 50 00 PM

Playground (no content):

Screen Shot 2021-09-27 at 1 50 39 PM

Playground (with content):

Screen Shot 2021-09-27 at 1 50 52 PM

## Links

JIRA: https://codedotorg.atlassian.net/browse/CSA-595

## Testing story

Tested on all the things level on Chrome, various window sizes.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/PlaygroundVisualizationColumn.jsx [hunk 1]
```diff
@@ -77,7 +77,6 @@ class PlaygroundVisualizationColumn extends React.Component {
 
 const styles = {
   playground: {
-    backgroundColor: 'white',
```
Comment (molly-moen) [inline apps/src/javalab/PlaygroundVisualizationColumn.jsx:80]: if a playground is created without a background now, what will it look like? I think we still want the default white background once the playground is started

Comment (sanchitmalhotra126) [reply apps/src/javalab/PlaygroundVisualizationColumn.jsx:80]: Hmm yeah it'll have no background, i.e. just the preview image - I'll add logic to playground to set the default white background on start. I guess we'd need to do the same for theater too?

Comment (molly-moen) [reply apps/src/javalab/PlaygroundVisualizationColumn.jsx:80]: I'm not sure about theater, I think we might set a default white background on the backend.

Comment (sanchitmalhotra126) [reply apps/src/javalab/PlaygroundVisualizationColumn.jsx:80]: Yeah, theater has a default white background so I've added this to just playground.

**Review by molly-moen — APPROVED** [review]
LGTM!

**Pull request merged**